### PR TITLE
Management eol of the systray

### PR DESCRIPTION
**Pull request by vladak**

### CODE REVIEW

**Comment by tarzanek** [comment]
Tests in error: 

  RuntimeEnvironmentTest.testProjects:144 » NullPointer
?

**Comment by vladak** [comment]
yep, trying to find the cause of the failing test.

**Comment by vladak** [comment]
seems like the removal of management from `tools/findbugs_filter.xml` did the trick.

File: README.txt [hunk 1]
```diff
@@ -13,7 +13,6 @@ OpenGrok - a wicked fast source browser
 5.  OpenGrok setup
 6.  Optional Command Line Interface Usage
 7.  Change web application properties or name
-8.  OpenGrok systray
```
Comment (tulinkry) [inline README.txt]: copyright

Comment (vladak) [inline README.txt]: fixed

**Comment by tarzanek** [comment]
merging, thank you!

**Pull request merged**